src/Modbus.Common/Util/Extensions.cs

```diff
@@ -69,8 +69,9 @@ public static ushort GetUInt16(this ModbusObject register)
 		/// </summary>
 		/// <param name="list">The list of registers (min. 2).</param>
 		/// <param name="startIndex">The start index. Default: 0.</param>
+		/// <param name="bigEndianRegistryOrder">Change the order of WORDs in multi-registry data. True indicates Big Endian encoding, False Little Endian encoding. Default: True</param>

```

Comment:
```suggestion
		/// <param name="inverseRegisters">Inverses the register order as required by some implementations. Default: false.</param>

```

That name is easier to understand what happens with this flag.